src/radical/pilot/agent/bootstrap_1.sh

```diff
@@ -80,7 +80,7 @@ LOCK_TIMEOUT=600 # 10 min
 VIRTENV_TGZ_URL="https://pypi.python.org/packages/source/v/virtualenv/virtualenv-1.9.tar.gz"
 VIRTENV_TGZ="virtualenv-1.9.tar.gz"
 VIRTENV_IS_ACTIVATED=FALSE
-VIRTENV_RADICAL_DEPS="pymongo==2.8 apache-libcloud colorama python-hostlist ntplib pyzmq netifaces==0.10.4 setproctitle orte_cffi msgpack-python future"
+VIRTENV_RADICAL_DEPS="colorama pymongo==2.8 apache-libcloud colorama python-hostlist ntplib pyzmq netifaces==0.10.4 setproctitle orte_cffi msgpack-python future"
```

Comment:
Yes indeed. I added because I was getting an error that there was no such module. Let me give it another try, because this is weird.

Is there a chance that the order is important?